scr/src/main/java/org/apache/felix/scr/impl/ComponentRegistry.java

```diff
@@ -560,7 +562,7 @@ public <T> void leaveCreate(final ServiceReference<T> serviceReference)
      * @param serviceReference
      * @param actor
      */
-    public synchronized <T> void missingServicePresent( final ServiceReference<T> serviceReference, ComponentActorThread actor )
+    public synchronized <T> void missingServicePresent( final ServiceReference<T> serviceReference, ScheduledExecutorService actor )
```

Comment:
We should stop passing the `actor` here since it is the same object passed to the constructor and stored in `m_componentActor`.